tests/test_mlflow.py

```diff
@@ -0,0 +1,80 @@
+import json
+import subprocess
+import tempfile
+from pathlib import Path
+
+import pytest
+from sklearn.base import BaseEstimator
+
+mlflow = pytest.importorskip("mlflow")
```

Comment:
This doesn't run for CI because itll skip right? 

Is there a way to make a github workflow like we did with airflow?
https://github.com/LineaLabs/lineapy/blob/main/.github/workflows/airflow.yml

The test skipping locally makes sense, but it makes me kind of nervous if CI doesn't run it if you make changes to mlflow files.

Reply:
Feel free to punt this for another ticket and PR, but this absolutely should be documented if the test no-ops in CI automatically.

Reply:
Add a new ticket https://linea.atlassian.net/browse/LIN-707